test/datasets/test_wikitexts.py

```diff
@@ -0,0 +1,94 @@
+import os
+import random
+import string
+import zipfile
+from collections import defaultdict
+from unittest.mock import patch
+
+from ..common.parameterized_utils import nested_params
+from torchtext.datasets.wikitext103 import WikiText103
+from torchtext.datasets.wikitext2 import WikiText2
+
+from ..common.case_utils import TempDirMixin, zip_equal
+from ..common.torchtext_test_case import TorchtextTestCase
+
+
+def _get_mock_dataset(root_dir, base_dir_name):
+    """
+    root_dir: directory to the mocked dataset
+    base_dir_name: WikiText103 or WikiText2
+    """
+    base_dir = os.path.join(root_dir, base_dir_name)
+    temp_dataset_dir = os.path.join(base_dir, "temp_dataset_dir")
+    os.makedirs(temp_dataset_dir, exist_ok=True)
+
+    seed = 1
+    mocked_data = defaultdict(list)
+    file_names = ("wiki.train.tokens", "wiki.valid.tokens", "wiki.test.tokens")
+    for file_name in file_names:
+        csv_file = os.path.join(temp_dataset_dir, file_name)
+        mocked_lines = mocked_data[os.path.splitext(file_name)[0]]
+        with open(csv_file, "w") as f:
+            for i in range(5):
+                rand_string = " ".join(
+                    random.choice(string.ascii_letters) for i in range(seed)
+                )
+                dataset_line = (rand_string,)
```

Comment:
Each element is actually a string, for both datasets. _E.g._ 

> ' = = Filmography = = \n'

Reply:
> Each element is actually a string, for both datasets. _E.g._
> 
> > ' = = Filmography = = \n'

I'm so sorry I mistyped there. I meant to say can we make each element a string instead of a tuple. `dataset_line = (rand_string,)` creates a tuple correct?

Reply:
Got it, just changed to string.